tools/build/scripts/extract-api-summary.js

```diff
@@ -28,18 +28,30 @@ if (undefined === argv.outDir) {
 
 fs.ensureDir(path.normalize(argv.outDir));
 
+let shouldGenerateFullReport = false;
+if (undefined !== argv.gatherFullReport)
+  shouldGenerateFullReport = true;
+
 // create output file
 const trimmedApiSignature = (argv.apiSignature.split('.'))[0];
 const sigFileName = path.basename(path.normalize(trimmedApiSignature));
-const sigFilePath = path.join(argv.outDir, sigFileName + ".exports.csv");
-fs.createFileSync(sigFilePath);
+const sigFilePath = path.join(argv.outDir, `${shouldGenerateFullReport ? "summary" : sigFileName}.exports.csv`);
 
 const outputLines = [];
-outputLines.push("sep=;");
-outputLines.push("Release Tag;API Item");
+if (shouldGenerateFullReport) {
+  if (fs.existsSync(sigFilePath))
+    outputLines.push("");
+  else {
+    outputLines.push("sep=;");
+    outputLines.push("Package Name;Release Tag;API Item");
+  }
+} else if (!shouldGenerateFullReport) {

```

Comment:
```suggestion
} else {
```

Reply:
Thanks, I previously had the other if statement merged with the first one.. Doesn't make sense anymore though.